src/main/java/org/mozilla/magnet/scanner/BaseScanner.java

```diff
@@ -1,8 +1,5 @@
-package org.mozilla.magnet.net.scanner;
```

Comment:
I don't regard bluetooth scanning as 'network' scanning. Namespacing `scanner` suggests that we may someday have more than one android scanner package. I don't think this is likely.